### DISCUSSION TITLE
I need help with a RTLSDR-Airband on a PI 4

### DISCUSSION THREAD

**Discussion by VE3TRU** [rtl-airband/RTLSDR-Airband#256] [Q&A]
I need some help I'm using a Pi 4 and I'm getting this error
pi@raspberrypi:~ $ sudo rtl_airband -f
Detached kernel driver
Found Rafael Micro R820T tuner
[R82XX] PLL not locked!
Allocating 10 zero-copy buffers 

Thanks

**Comment by szpajder** [comment]
You mean `PLL not locked`?

I see it pretty often, but I don't observe any adverse effects. Just ignore it.

**Reply by VE3TRU** [reply]
Its not working at all, and It's a PI 4, maybe that has something to do with it IDK. I followed these instructions.
https://fuzzthepiguy.tech/rtlsdr-airband/

**Reply by szpajder** [reply]
My crystal ball is temporarily out of service.

1. Read the documentation: https://github.com/szpajder/RTLSDR-Airband/wiki.
2. Make a simple config file based on one of the example configs supplied by the program.
3. Start slowly - multichannel mode, single channel, verify if it works. Only then add more channels or use other modes.
4. Run the program in foreground - you'll see textual waterfalls and you will know what's going on
5. If you encounter problems - read the logs, check if there are any error messages or warnings produced by the program
6. Verify that your setup is actually working, using some other software (GQRX, SDRSharp, SDR Console, whatever). If your antenna is not connected or your USB cable is broken - how can anybody guess that?
7. If it still doesn't work - come back here, but describe the problem precisely. Given the problem described with "Its not working at all" I can only tell that probably something is wrong.

**Reply by VE3TRU** [reply]
I'm using only the one SDR dongle, and it works fine with SDR# on my windows machine.
Its suppose to scan through the freq in the terminal, and it doesn't its a terminal type program. Also Icecsat doesn't recognize it
Here is how my config file looks.


pi@raspberrypi:~ $ sudo nano /usr/local/etc/rtl_airband.conf
  GNU nano 3.2            /usr/local/etc/rtl_airband.conf

# Each channel is sent to a single Icecast output.
# Refer to https://github.com/szpajder/RTLSDR-Airband/wiki
# for description of keywords and config syntax.

devices:
({
type = "rtlsdr";
index = 0;    #RTL-SDR Number
gain = 32;    #Gain Setting
correction = 0;
mode = "scan";
channels:
(
{
freqs = ( 123.1, 119.0, 120.5 );        #Add Freq's Here
#labels = ( "Tower A", "Tower B", "Tower Control"); #Label Update
outputs: (
{
type = "icecast";
server = "127.0.0.1";
port = 8000;   #Port Number Setting
mountpoint = "stream.mp3";
name = "Airband Voice";   #Name Of Your Choice
genre = "Air Traffic Control";
description = "My Local Air Traffic"; #Description Of Your Choice
username = "source";
password = "raspberry";   #Icecast2 Password
send_scan_freq_tags = false;
}
);
}
);
}
);

**Reply by LoganSound** [reply]
if you followed that tutorial exactly you compiled for RPiv2, needs to compile with:
make PLATFORM=armv8-generic
For RPi4

**Reply by VE3TRU** [reply]
Thanks I had a feeling that way the tutorial was with a PI v2 and I'm using a PI v4 I haven't tried it yet will do so.
